docs/software/cloud.md

```diff
@@ -962,8 +963,38 @@ Rclone is a command-line program to manage files on cloud storage. It is a featu
 
 ***
 
-Official documentation: <https://rclone.org/>  
-Source code: <https://github.com/rclone/rclone>  
+Official documentation: <https://rclone.org/>
+Source code: <https://github.com/rclone/rclone>
```

Comment:
The spaces need to stay here, otherwise there will be no line breaks:
```suggestion
Official documentation: <https://rclone.org/>  
Source code: <https://github.com/rclone/rclone>  
```